integration/feature/init/src/MillInitSbtTests.scala

```diff
@@ -0,0 +1,198 @@
+  def tests: Tests = Tests {
+    /*
+    - 12 KB
+    - originally sbt 1.0.2
+     */
+    val url =
+      "https://github.com/pbassiner/sbt-multi-project-example/archive/152b31df9837115b183576b0080628b43c505389.zip"
+
+    test - integrationTest(url) { tester =>
+      import tester.*
+      bumpSbtTo1107(workspacePath)
+      /*
+      `multi1.compile` doesn't work well when Mill is run with JDK 17 and 21:
+      ```text
+      1 tasks failed
+      multi1.compile java.io.IOError: java.lang.RuntimeException: /packages cannot be represented as URI
+          java.base/jdk.internal.jrtfs.JrtPath.toUri(JrtPath.java:175)
+          scala.tools.nsc.classpath.JrtClassPath.asURLs(DirectoryClassPath.scala:183)
+          ...
+      ```
+      Passing a `jvmId` 11 doesn't work here.
+       */
+      writeMillJvmVersionTemurin11(workspacePath)
+
+      val submodules = Seq("common", "multi1", "multi2")
+      testMillInit(
+        tester,
+        expectedCompileTasks = Some(SplitResolvedTasks(
+          successful = Seq("compile") ++ submodules.flatMap(allCompileTasks),
+          failed = Seq.empty
+        )),
+        expectedTestTasks =
+          Some(SplitResolvedTasks(successful = submodules.map(testTask), failed = Seq.empty))
+      )
+    }
+  }
+}
+
+// relatively large libraries
+
+object MillInitSbtGatlingTests extends BuildGenTestSuite {
+  def tests: Tests = Tests {
+    /*
+    - 1.8 MB
+    - sbt 1.10.7
+     */
+    val url = "https://github.com/gatling/gatling/archive/refs/tags/v3.13.4.zip"
+
+    val submodules = Seq(
+      "gatling-app",
+      "gatling-benchmarks",
+      "gatling-charts",
+      "gatling-commons",
+      "gatling-core-java",
+      "gatling-core",
+      "gatling-http-client",
+      "gatling-http-java",
+      "gatling-http",
+      "gatling-jdbc-java",
+      "gatling-jdbc",
+      "gatling-jms-java",
+      "gatling-jms",
+      "gatling-jsonpath",
+      "gatling-netty-util",
+      "gatling-quicklens",
+      "gatling-recorder",
+      "gatling-redis-java",
+      "gatling-redis",
+      "gatling-samples",
+      "gatling-test-framework"
+    )
+    val submodulesWithoutTests = Seq(
+      "gatling-app",
+      "gatling-benchmarks",
+      "gatling-quicklens",
+      "gatling-samples",
+      "gatling-test-framework"
+    )
+    val submodulesWithTests = submodules diff submodulesWithoutTests
+
+    test - integrationTest(url) { tester =>
+      testMillInit(
+        tester,
+        expectedCompileTasks = Some(SplitResolvedTasks(
+          all = Seq("compile")
+            ++ submodulesWithTests.flatMap(allCompileTasks)
+            ++ submodulesWithoutTests.map(compileTask),
+          failed = Seq(
+            /*
+            `...gatling-benchmarks/src/main/scala/io/gatling/Utils.scala:26:19: no arguments allowed for nullary method toString: (): String`
+            This fails to compile with sbt too. See https://github.com/gatling/gatling/issues/4612.
```

Comment:
Looks like this was fixed! https://github.com/gatling/gatling/commit/40094059e644978624648c261e20723eb05afba7 we can pull it in the latest version and remove this special case

Reply:
It's not released yet though. Shall I change the version tag to a commit in the url to achieve this?